src/app/core/market/order-status-notifier/order-status-notifier.service.ts

```diff
@@ -19,78 +24,101 @@ export class OrderStatusNotifierService implements OnDestroy {
             'Order rejected'
           ];
   public oldOrders: Array<any>;
-
+  orders: Bid;
```

Comment:
This is supposed to be an array of Bids.
The Bid model is really mangled, it is supposed to be _one_ bid and not a collection of _bids_.